### PR TITLE
The description and example belong to the item and not to the array

### PR DESCRIPTION
**Pull request by tmuehl**
@v-stepanov @rcillo @jkliff @antban

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/237" target="_blank">ReviewNinja

**Comment by rcillo** [comment]
👍 thanks for the contribution.

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -901,10 +901,10 @@ definitions:
...
               type: string
               format: uuid
-            description: |
-              Event identifier of the Event that caused the generation of this Event.
-              Set by the producer.
-            example: '105a76d8-db49-4144-ace7-e683e8f4ba46'
+              description: |
+                Event identifier of the Event that caused the generation of this Event.
+                Set by the producer.
+              example: '105a76d8-db49-4144-ace7-e683e8f4ba46'
```
Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: I know that it wasn't affected by you, but shouldn't the example be an array?

Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: I don't think so because the example is for the item.

Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: Ahhh ok. You actually moved it to the right, so it's now under "items". Good.

**Pull request merged**